largest_sum_contiguous_subarray/LargestSumContiguousSubarray.java

```diff
@@ -0,0 +1,39 @@
+/* 
+   *   Implementation of famous dynamic programming problem
+   *   Largest Sum Contiguous Subarray
+   *   Kadane Algorithm 
+   *   Time complexity  O(n)
+*/
+package largestsum;
+
+public class LargestSumContiguousSubarray {
+
+	private static void maxSumSubArray(int[] array) {   //  maximum sum method implemention
+		int prevSum;
+		int currentSum;
+		int i;
+		prevSum = array[0];						// initialize current sum amd previous sum
+		currentSum = array[0];
+
+		for(i = 1; i < array.length; i++){
+			currentSum += array[i];				// add values in current sum
+			if(currentSum < 0){					// if current sum is negative , make it zero
+				currentSum = 0;
+			}else{
+				if (currentSum > prevSum) {		// if current sum is greate than previous sum
+					prevSum = currentSum;			// update previous sum
+				}
+			}
+		}
+		System.out.println("Maximum Sum of Subarray:" + "\t" + prevSum);
```

Comment:
No prints in algorithm function. Toss back value in main and print result(s) there.